<img src="http://willfreyman.org/assets/img/onag_big1.jpg" width="350" />

# IB290: Topics in Phylogenetics
# UC Berkeley

* Grad Seminar 460
* Class Number 67082
* Fall, 2019
* Thursdays 10:10 - 11:00 AM (Berkeley time!!!)
* 1002 VLSB
* Class website: [https://mikeryanmay.github.io/IB290/](https://mikeryanmay.github.io/IB290/)

## Instructors

* Carl Rothfels -- crothfels@berkeley.edu
* John Huelsenbeck -- johnh@berkeley.edu
* Mike May -- mrmay@berkeley.edu

## Format

The instructors will lead a series of introductory sessions (lectures) for the first four or five meetings. Subsequent meetings will be student-led. Each student will be responsible for leading at least one session (potentially more, depending on enrollment and interest) on a topic of their choice, in consultation with the instructors. That student will be responsible for developing an annotated bibliography of the important papers on the topic, including both “classics” and the most recent works, for selecting two or so of the papers for the class to read, and for preparing a short (15min) presentation on the topic. The remainder of the session will be devoted to discussion.

## Course Texts

There are no formal texts required for this seminar, but participants are encouraged to consult *Tree Thinking: An Introduction to Phylogenetic Biology* (Baum & Smith 2012) and also *Inferring Phylogenies* (Felsenstein 2004) and *Molecular Evolution: A Statistical Approach* (Yang 2014).

## Useful Info

Here are some useful links on statistics and phylogenetic software: [https://mikeryanmay.github.io/IB290/useful](https://mikeryanmay.github.io/IB290/useful)

## Possible Topics for Student Led Discussions

*Please consult an instructor before settling on a topic:*

modeling migration/hybridization/introgression (phylogenetic networks); state-dependent diversification models; fossil data (e.g. fossilized-birth-death process/tip dating vs node calibrations); modeling gene duplication and loss; polymorphism-aware (PoMo) models; covarion/hidden state models; model testing (e.g. Bayes factors, AIC); model adequacy (e.g. posterior predictive tests); demographic inference (e.g., inferring population size changes through time); the multispecies coalescent, genetrees/species tree; the BAMM controversy; divergence time dating; inferring selection from sequence alignments; alignment inference (alignment-phylogeny co-inference?); community phylogenetics; spatial phylogenetics; inferring phylogenies from morphological data; heterogeneous models, e.g., non-stationary base frequencies; ancestral state reconstruction; phylogenomics (strengths, pitfalls?); inferring correlated evolution of traits; assessing support; hypothesis testing of relationships; reconstructing morphological evolution on a phylogeny; ABC approaches; hidden Markov models; mixture models; species delimitation; microbial community analysis; quartet-based methods (e.g. SVDquartets); etc; etc;

When you have chosen a topic, update this README by following these instructions: [updating the course website](https://mikeryanmay.github.io/IB290/git_workflow)


## Schedule:

Our first meeting will be on Thursday, September 5.

### September 5: Introductions

*Who are we, and want to do we want to get out of this seminar?*
* Outline of the course
* Outline of “phylogenetics”, i.e., introduction to trees—what is a phylogeny?—and tree thinking—what does this tree say?
* Discussion of topics to select for future class sessions
* Introduction to Git
* Reading: chapters 1, 2, and 3 of Tree Thinking
* Slides: [introduction to git](https://mikeryanmay.github.io/IB290/lecture_slides/git.pdf)

<!-- * Slides: [Rothfels Intro Slides](https://wf8.github.io/IB290/lecture_slides/Mtg01_intro.ppt) -->

### September 12: Introduction to probability theory

*Introduction to probability, estimation, and inference:*
* Maximum likelihood
* Bayesian estimation
* MCMC
* Reading:
* Slides:

<!-- * Reading: [Coin Tossing - Diaconis et al 2007](https://wf8.github.io/IB290/readings/2007_Diaconis.pdf) -->

### September 19: Introduction to tree inference and Markov models of character change

*Introduction to the application of Markov models to tree inference (Part 1):*
* What is a Markov model?
* Properties and uses of Markov models
* Pruning algorithm
* Reading:
* Slides:

<!-- * Slides: [Huelsenbeck Likelihood Phylo Slides](https://wf8.github.io/IB290/lecture_slides/Mtg02_Huelsenbeck_Likelihood_Phylo.pdf) -->

### September 26: Introduction to tree inference and Markov models of character change

*Introduction to the application of Markov models to tree inference (Part 2):*
* More on Markov models
* Substitution models
* Reading:
* Slides:

<!-- * Slides: [Huelsenbeck Likelihood Phylo Slides](https://wf8.github.io/IB290/lecture_slides/Mtg03_Huelsenbeck_Likelihood_Phylo.pdf) -->

### October 3: Introduction to "tree models" and their uses

*Introduction to models of tree shape, topology, and branch lengths:*
* Branching-process models and their application to:
  * Divergence time estimation
  * Diversification rate estimation
* Reading:
* Slides:


### October 10: Phylogenetics potpourri -- recap and misc. stuff that we didn't cover yet

TBD


### October 17: TBD

* Students:
* Reading:
* Annotated bibliography:

### October 24: TBD

* Students:
* Reading:
* Annotated bibliography:

### October 31: A spooky TBD

* Students:
* Reading:
* Annotated bibliography:

### November 7: TBD

* Students:
* Reading:
* Annotated bibliography:

### November 14: TBD

* Students:
* Reading:
* Annotated bibliography:

### November 21: TBD

* Students:
* Reading:
* Annotated bibliography:


### December 5: TBD

* Students:
* Reading:
* Annotated bibliography:

### December 12: TBD

* Students:
* Reading:
* Annotated bibliography:
